db/global.go

```diff
@@ -0,0 +1,79 @@
+package db
+
+// NewGlobal constructs a new global object.
+func NewGlobal() *Global {
+	return &Global{}
+}
+
+// Path returns the path to the global configuration
+func (g *Global) Path() (string, error) {
+	return rootGlobal, nil
+}
+
+// Prefix returns the path to the global configuration
```

Comment:
The prefix is a path to itself; this is an appropriate word choice.

On 15 Aug 2016, at 23:47, Vikrant Balyan wrote:

>> @@ -0,0 +1,79 @@
>> +package db
>> +
>> +// NewGlobal constructs a new global object.
>> +func NewGlobal() *Global {
>> +	return &Global{}
>> +}
>> +
>> +// Path returns the path to the global configuration
>> +func (g *Global) Path() (string, error) {
>> +	return rootGlobal, nil
>> +}
>> +
>> +// Prefix returns the path to the global configuration
>
> "path" -> "prefix"
>
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/408/files/1a3157f1b1181a25f0c3c2e5792ffecb6ca6e07d#r74883222